v03_pipeline/var/test/pedigrees/test_pedigree_6.tsv

```diff
@@ -4,4 +4,5 @@ R0116_sex_check_project2	family_1	family_1	ROS_006_18Y03227_D1			U
 R0116_sex_check_project2	family_1	family_1	ROS_006_18Y03228_D1	ROS_006_18Y03226_D1	ROS_006_18Y03227_D1	F
 R0116_sex_check_project2	family_2	family_2	ROS_007_19Y05919_D1			F
 R0116_sex_check_project2	family_2	family_2	ROS_007_19Y05939_D1			F
-R0116_sex_check_project2	family_2	family_2	ROS_007_19Y05987_D1	ROS_007_19Y05919_D1	ROS_007_19Y05939_D1	F
\ No newline at end of file
+R0116_sex_check_project2	family_2	family_2	ROS_007_19Y05987_D1	ROS_007_19Y05919_D1	ROS_007_19Y05939_D1	F
+R0116_sex_check_project2	family_2	family_2	ROS_007_19Y05989_D1	ROS_007_19Y05919_D1	ROS_007_19Y05939_D1	XXX
```

Comment:
need to confirm that pedigree file will also be delivered with sex aneupolodies matching those in the imputed sex file predicted_sex

Reply:
confirmed! This is how seqr would generate a pedigree file with an aneuploidy